### ISSUE TITLE
Hypertables and Chunk support

### ISSUE THREAD

**Issue by davydog187** [bitfo/timescale#3]
Add support for create and managing hypertables and chunks

- [x] [create_hypertable](https://docs.timescale.com/api/latest/hypertable/create_hypertable/)
- [ ] [show_chunks](https://docs.timescale.com/api/latest/hypertable/show_chunks/)
- [ ] [drop_chunks](https://docs.timescale.com/api/latest/hypertable/drop_chunks/)
- [ ] [reorder_chunk](https://docs.timescale.com/api/latest/hypertable/reorder_chunk/)
- [ ] [move_chunk](https://docs.timescale.com/api/latest/hypertable/move_chunk/)
- [ ] [add_reorder_policy](https://docs.timescale.com/api/latest/hypertable/add_reorder_policy/)
- [ ] [remove_reorder_policy](https://docs.timescale.com/api/latest/hypertable/remove_reorder_policy/)
- [ ] [attach_tablespace](https://docs.timescale.com/api/latest/hypertable/attach_tablespace/)
- [ ] [detach_tablespace](https://docs.timescale.com/api/latest/hypertable/detach_tablespace/)
- [ ] [detach_tablespaces](https://docs.timescale.com/api/latest/hypertable/detach_tablespaces/)
- [ ] [show_tablespaces](https://docs.timescale.com/api/latest/hypertable/show_tablespaces/)
- [ ] [set_chunk_time_interval](https://docs.timescale.com/api/latest/hypertable/set_chunk_time_interval/)
- [ ] [set_integer_now_func](https://docs.timescale.com/api/latest/hypertable/set_integer_now_func/)
- [ ] [add_dimension](https://docs.timescale.com/api/latest/hypertable/add_dimension/)
- [ ] [create_index (transaction per chunk)](https://docs.timescale.com/api/latest/hypertable/create_index/)
- [ ] [hypertable_size](https://docs.timescale.com/api/latest/hypertable/hypertable_size/)
- [ ] [hypertable_detailed_size](https://docs.timescale.com/api/latest/hypertable/hypertable_detailed_size/)
- [ ] [hypertable_index_size](https://docs.timescale.com/api/latest/hypertable/hypertable_index_size/)
- [ ] [chunks_detailed_size](https://docs.timescale.com/api/latest/hypertable/chunks_detailed_size/)

**Comment by doomspork** [comment]
@davydog187 do you want to add these functions to the `migration.ex` file or start to break things up but grouping (ie `chunks.ex`) and import them?

**Comment by davydog187** [comment]
What if we added to migration, but then for the more diagnostic functions we add to a different module?

**Comment by davydog187** [comment]
I don't think we need most of these for the first release. Will split this out
